common/notification/actions.ts

```diff
@@ -618,6 +618,10 @@ const _notificationActions = {
     // ACHIEVEMENT TEST ACTIONS
 
     /* ONBOARDING */
+    reward_issued: {
```

Comment:
maybe rename to `user_achievement_rewarded` to make it more clear

Reply:
user_achievemnet_reward_issued